session/auth/authprovider/authprovider.go

```diff
@@ -29,6 +29,8 @@ import (
 )
 
 const defaultExpiration = 60
+const dockerIndexConfigfileKey = "https://index.docker.io/v1/"
+const dockerRegistryHost = "registry-1.docker.io"
```

Comment:
small nit, these vars should have `Hub` in the name. eg.

```
dockerHubRegistryHost
dockerHubConfigfileKey
```

Reply:
done!